pkgs/by-name/ni/nil/package.nix

```diff
@@ -20,9 +20,7 @@ rustPlatform.buildRustPackage rec {
   useFetchCargoVendor = true;
   cargoHash = "sha256-7TR/xTc66WpPszJDrpisVvHXl2+FGrUOskZAkGyY04Q=";
 
-  nativeBuildInputs = [
-    (lib.getBin nixVersions.latest)
```

Comment:
@oxalica please confirm we no longer need this with nix 2.24.

Reply:
Maybe @figsoda would also be able to confirm this?

Reply:
Figsoda is sadly not so active lately.

Reply:
Ah ok. I hope he's well.

I'm willing to assume the removal is ok, considering that it's a build time dependency, and it builds, but let's wait until Monday to give us an opportunity to handle this issue without potential for a minor(?) regression.

Reply:
> @oxalica please confirm we no longer need this with nix 2.24.

Sorry for the late response. As mentioned by @roberth, yes this is a build-time dependency to populate the builtin function list. As long as it builds, it's good.